src/teacher/StudentsActivityOverview.js

```diff
@@ -20,7 +20,14 @@ export default function StudentsActivityOverview({ api }) {
   const [showAddStudentsInfo, setShowAddStudentsInfo] = useState(false);
   const history = useHistory();
 
-  //Extracting the cohort data for the page title, for showing no student guidance and for deleting students from the cohort.
+  const getStudentsToShow = () =>{
```

Comment:
why do you like this more than 

```

    function getStudentsToShow() {
}
```
this latter one is shorter to write, and more clear - you're defining a function not a constant :)

Reply:
also a rename would be nice that highlights that the function not only "gets students to show" but also "shows them"!

i was looking at the following snippet and was wondering: why only get students? on remove? turns out it's not only *get* but also at least a *set* which implies an update; which makes sense :)
![image](https://user-images.githubusercontent.com/464519/125272811-ef629b00-e314-11eb-94e2-f7eed4d65718.png)

Reply:
Fixed in my next push to mvp